jwst/saturation/saturation_step.py

```diff
@@ -16,6 +16,11 @@ class SaturationStep(Step):
 
     class_alias = "saturation"
 
+    spec = """
+        n_pix_grow_sat = int(default=1) # number of layers adjacent pixels to flag
+    """
```

Comment:
Will need to now pass `self.n_pix_grow_sat` as an argument to both `irs2_flag_saturation` and `flag_saturation`, so that it eventually makes its way down to the low-level functions.